src/6_branch.js

```diff
@@ -1654,11 +1657,16 @@ Branch.prototype['banner'] = wrap(callback_params.NO_CALLBACK, function(done, op
 		bannerOptions.showKindle = options['showMobile'];
 	}
 
+	this.setBranchViewData(data);
 	this.closeBannerPointer = banner(this, bannerOptions, data, this._storage);
 	done();
 });
 
-Branch.prototype['closeBanner'] = wrap(0, function(done) {
```

Comment:
makes sense - will the behavior/ availability of 'closeBanner' be otherwise affected at all in this merge?